src/click/_termui_impl.py

```diff
@@ -62,6 +62,7 @@ def __init__(
         label=None,
         file=None,
         color=None,
+        update_interval=1,
```

Comment:
Maybe call this `update_min_steps` to make it clearer what the interval is and how it's checked.

Reply:
Done in d3c5e05. I named it before jumping into the implementation, but I agree that that's a better name.